### PR TITLE
fix: defaultEventLocation's icon color in dark mode

### LINKED ISSUE #10213: [CAL-2196] location icon missing dark mode
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2196](https://linear.app/calcom/issue/CAL-2196/location-icon-missing-dark-mode)</sub>

### PR DESCRIPTION
**Pull request by anikdhabal**
## What does this PR do?

Fixes #10213 

## Type of change

  - [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/features/bookings/components/event-meta/Locations.tsx [hunk 1]
```diff
@@ -21,9 +21,10 @@ export const EventLocations = ({ event }: { event: PublicEvent }) => {
   };
   const eventLocationType = getEventLocationType(locations[0].type);
   const icon = (locations.length > 1 || !eventLocationType?.iconUrl) ? MapPin : eventLocationType.iconUrl;
+  const isDark = !eventLocationType?.iconUrl?.startsWith("/app-store") ? true : false;
```
Comment (PeerRich) [inline packages/features/bookings/components/event-meta/Locations.tsx:24]: you dont need ? true : false for boolean statements

Comment (anikdhabal) [reply packages/features/bookings/components/event-meta/Locations.tsx:24]: yes yes sorry my mistake. Let me fix it.

**Comment by PeerRich** [comment]
already fixed in https://github.com/calcom/cal.com/pull/9993 thank you for the help tho!

**Comment by anikdhabal** [comment]
> already fixed in #9993 thank you for the help tho!

But this problem is not solving yet.
This is the result after i fix it.

**Comment by anikdhabal** [comment]
> already fixed in #9993 thank you for the help tho!

okay

**Pull request closed without merging**